components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -4507,19 +4554,19 @@ def __eq__(self, a):
         :return:    True if annotations are the same - see above
         :rtype:     Boolean
         """
-        return (type(a) == type(self) and self._obj.id == a._obj.id
-                and self.getValue() == a.getValue()
-                and self.getNs() == a.getNs())
+        return (type(a) == type(self) and self._obj.id == a._obj.id and
+                self.getValue() == a.getValue() and
+                self.getNs() == a.getNs())
 
-    def _getQueryString(self):
+    def _getQueryString(self, child_count=False):
         """
         Used for building queries in generic methods such as
         getObjects("Annotation")
         """
-        return ("select obj from Annotation obj "
-                "join fetch obj.details.owner as owner "
-                "join fetch obj.details.group "
-                "join fetch obj.details.creationEvent")
+        query = ("select obj from Annotation obj "
+                 "join fetch obj.details.owner as owner "
```

Comment:
why? we need one at least for parent. You could parametrize the test. In my opinion well written test is much better then the doc